src/yardstick/cli/config.py

```diff
@@ -65,11 +66,31 @@ def __post_init__(self):
         self.images = images
 
 
+@dataclass()
+class Validation:
+    name: str = "default"
+    max_f1_decrease: float = 0.0
+    max_unlabeled_match_percent: int = 0
+    max_new_false_negatives: int = 0
+
+
 @dataclass()
 class ResultSet:
     description: str = ""
     declared: list[artifact.ScanRequest] = field(default_factory=list)
     matrix: ScanMatrix = field(default_factory=ScanMatrix)
+    max_year: int | None = None
+    validations: list[Validation] = field(default_factory=list)
+
+    def tool_comparisons(self) -> tuple[str, str]:
+        reference, candidate = "", ""
+        for t in self.matrix.tools:
+            if t.validation_role == "candidate":
+                candidate = f"{t.name}@{t.version}"
```

Comment:
This is supposed to return the same string as the tool version that's in `.tools` on a ScanResult, but nothing enforces this.